packages/parts/src/lib/maps/utils/dictionaryGeoJsonCollection.ts

```diff
@@ -11,7 +11,9 @@ import type { GeoJSONSourceOptions } from 'mapbox-gl';
 //   CentralAmerica: [-80, 5],
 // };
 
-export function dictionaryGeoJsonCollection(dictionaries: IDictionary[]): GeoJSONSourceOptions['data'] {
+export function dictionaryGeoJsonCollection(
+  dictionaries: IDictionary[] | Partial<IDictionary>[]
```

Comment:
@jacob-8 here's where I have more doubts, I think is not correct to add the Partial<IDictionary> type.
Should I make another function that only retrieves the coordinates for each place where a language is spoken? Not only the main coordinates?